topics/02_Controls/igGrid/03_Features/09_igGrid_jsRender_Integration.md

```diff
@@ -212,8 +245,10 @@ The following topics provide additional information related to this topic.
 ### <a id="samples"></a> Samples
 
 The following sample provides additional information related to this topic.
+<div class="embed-sample">
```

Comment:
Add the sample in the after the configuration steps as step 5 and remove the relates sample section.